ibverbs/src/lib.rs

```diff
@@ -1556,6 +1556,22 @@ impl<T> MemoryRegion<T> {
         self.data
     }
 
+    /// Make a lifetime-independent slice representation of this memory region.
+    /// Unlike `LocalMemorySlice`, this supports 64-bit memory region lengths.
+    pub fn as_slice(&self) -> MemoryRegionSlice {
+        let bounds = (..unsafe { *self.inner.mr }.length);
```

Comment:
these lines here seem obsolete, no need to recalculate the addr and length if we are taking the full range